ruby/bps-kafka/bps-kafka.gemspec

```diff
@@ -0,0 +1,22 @@
+Gem::Specification.new do |s|
+  s.name        = 'bps-kafka'
+  s.version     = File.read(File.expand_path('../.version', __dir__)).strip
+  s.platform    = Gem::Platform::RUBY
+
+  s.licenses    = ['Apache-2.0']
+  s.summary     = 'Kafka adapter for bps'
+  s.description = 'https://github.com/bsm/bps'
+
+  s.authors     = ['Dimitrij Denissenko']
```

Comment:
feel free to mention yourself :)

Reply:
I'd rather mention the whole team (I suppose, this `authors` thingy is kinda like "who to reach for support"?)